include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -511,16 +616,20 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 								}
 							}
 						}
+
+						value_type* phaseSupportLUTPixel = reinterpret_cast<value_type*>(state->scratchMemory + getPhaseSupportLUTByteOffset(state) + axisOffsets[axis]);
```

Comment:
this is a hot code path, please hoist the `getPhaseSupportLUTByteOffset(state)+axisOffsets[axis]`

(maybe pre-add to `axisOffsets`)

Reply:
Resolved in #343.